log/term/terminal_windows.go

```diff
@@ -8,15 +8,23 @@
 package term
 
 import (
+	"bytes"
+	"encoding/binary"
 	"io"
+	"regexp"
 	"syscall"
 	"unsafe"
+
+	"golang.org/x/sys/windows"
```

Comment:
Doesn't this come with Go itself ?

I mainly used that instead of reinventing the wheel but I guess I can reimplement those in there if you think it is better form.